cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -283,4 +281,10 @@ static void ensureHeaderNotPresent(HttpServletRequest request, String headerName
   public static String httpServletRequestToString(HttpServletRequest request) {
     return String.format("%s %s", request.getMethod(), request.getRequestURI());
   }
+
+  public static String getDefaultWebServerApiUrlPrefix() {
+    // Strip off the '*' from the end of the default web server API URL prefix
+    return WebServerConfig.DEFAULT_WEBSERVER_API_URLPREFIX
+        .substring(0, WebServerConfig.DEFAULT_WEBSERVER_API_URLPREFIX.length() - 1);
+  }
```

Comment:
After adding back the `SessionManager` class, it will use this method. So, I will leave it here for now and move it to some util test class when we remove `SessionManager`